### PR TITLE
[plugin.video.disclose_tv] 0.1.6

### PR DESCRIPTION
**Pull request by Spinalcracker**
Disclose.TV addon fixed and improved

fixed playback
defaults to HD streams
added setting to disable HD default
This PR as suggested by Martijn - see here: http://forum.kodi.tv/showthread.php?tid=147572&pid=2331972#pid2331972

.pyo files deleted and commits squashed

Thank you

### CODE REVIEW

**Comment by learningit** [comment]
Sorry that's not squashed. Pls see https://github.com/ginatrapani/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit for how to squash to a single commit.

**Comment by Spinalcracker** [comment]
Hmm, that didn't seem to squash these commits even though I have that as the only option in the repo settings on github.

Am I best to close this, refork and resubmit?

**Comment by learningit** [comment]
Did you do the force push?

**Comment by learningit** [comment]
You can use the same PR, just follow the instructions on the link I gave previously.

**Comment by Spinalcracker** [comment]
> You can use the same PR, just follow the instructions on the link I gave previously.

Ok, I'll try that tonight.  I was thinking you guys would just do the squash on the merge as per this documentation

https://github.com/blog/2141-squash-your-commits
https://help.github.com/articles/about-pull-request-merge-squashing/

I'm still learning all of this - thank you for your help.

**Comment by learningit** [comment]
We prefer a squashed, single commit because review of the changes is less error prone. We also don't have to guess about how to resolve potential conflicts of multiple commits, so the easy answer is to have a single commit.

**Comment by Spinalcracker** [comment]
I understand.  Makes sense.  I'll get it figured out tonight as there doesn't appear to be a way to do this from the website, and only have access from my phone atm.

**Comment by Spinalcracker** [comment]
Success!  A single commit.

**Comment by phil65** [comment]
One commit, but with a wrong title:)
Commit title should be: [plugin.video.disclose_tv] 0.1.6

**Comment by Spinalcracker** [comment]
Ok.. I think I got it finally, lol.  Thank you for your patience as I learn git.

File: plugin.video.disclose_tv/addon.xml [hunk 1]
```diff
@@ -0,0 +1,74 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="plugin.video.disclose_tv" name="Disclose.tv" version="0.1.6" provider-name="Sphere - patched by Spinalcracker :)">
+  <requires>
+    <import addon="xbmc.python" version="2.1.0"/>
```
Comment (phil65) [inline plugin.video.disclose_tv/addon.xml]: this needs to be 2.24.0

**Comment by phil65** [comment]
Please convert all strings.xml files to strings.po.

File: plugin.video.disclose_tv/addon.xml [hunk 2]
```diff
@@ -0,0 +1,74 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="plugin.video.disclose_tv" name="Disclose.tv" version="0.1.6" provider-name="Sphere - patched by Spinalcracker :)">
```
Comment (phil65) [inline plugin.video.disclose_tv/addon.xml]: Please change provider-name to "Sphere | Spinalcracker"

File: plugin.video.disclose_tv/resources/lib/scraper.py [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+
+    def __get_url(self, url):
+        log('__get_url opening url: %s' % url)
+        response = urlopen(url).read()
+        log('__get_url got %d bytes' % len(response))
+        return response
+
+
+def log(text):
+    print u'Scraper: %s' % text
```
Comment (phil65) [inline plugin.video.disclose_tv/resources/lib/scraper.py]: please dont print(), use xbmc.log() with debug level instead.

File: plugin.video.disclose_tv/addon.xml (hunk 2 above)
Comment (Spinalcracker) [inline plugin.video.disclose_tv/addon.xml]: Done

**Comment by phil65** [comment]
Still missing:
> Please convert all strings.xml files to strings.po.

After that it´s good to go I think.

**Comment by Spinalcracker** [comment]
> After that it´s good to go I think.

Ok, all strings.xml converted to strings.po

Thank you

**Comment by phil65** [comment]
Well, thanks from our side for contributing and your patience.

**Pull request merged**